cmd/ui/src/views/DownloadCollectors/DownloadCollectors.tsx

```diff
@@ -92,11 +92,10 @@ const DownloadCollectors = () => {
     /* Implementation */
     return (
         <PageWithTitle title='Download Collectors' data-testid='download-collectors'>
-            <Typography variant='body1'>
-                <p>To get started, collect data using SharpHound or AzureHound.</p>
-                <p>
-                    BloodHound CE supports both {sharpHoundCELink} or {azureHoundCELink} collectors.
-                </p>
+            <Typography variant='body2' paragraph={true}>
+                To get started, collect data using SharpHound or AzureHound.
+                <br />
+                BloodHound CE supports both {sharpHoundCELink} or {azureHoundCELink} collectors.
```

Comment:
Would it make more sense for this to read "BloodHound CE supports both {sharpHoundCELink} _**and**_ {azureHoundCELink} collectors." instead of using the `or` conjunction?

Reply:
Yup, got it!